packages/mds-db/types.ts

```diff
@@ -96,10 +96,12 @@ export interface VehicleEventCountResult {
 }
 
 export interface ReadGeographiesParams {
-  get_read_only: boolean
+  get_published?: boolean
+  get_unpublished?: boolean
+  geography_ids?: Array<UUID>
```

Comment:
Probably want this to be `UUID[]`, we don't use `Array<xxx>` in many other places.